src/graphql/query/flssLoansQuery.graphql

```diff
@@ -1,10 +1,10 @@
 #import '../fragments/loanCardFields.graphql'
 
 query flssLoanData(
-  $filterObject: [FundraisingLoanSearchFilterInput!] = { countryIsoCode: {any: ["US"]}}
+  $filterObject: [FundraisingLoanSearchFilterInput!]
   $imgDefaultSize: String = "w480h360"
   $imgRetinaSize: String = "w960h720"
-  $limit: Int = 20
+  $limit: Int = 50
```

Comment:
Please lower this default. This file will now become shared so we shouldn't be asking for 50 loans at a time by default.